docs/source/daml/intro/12_Testing.rst

```diff
@@ -36,6 +37,120 @@ There are three primary tools available in the SDK to test and interact with Dam
 
   If you want to do things interactively, Daml REPL is the tool to use. The best way to think of Daml REPL is as an interactive version of Daml Script, but it doubles up as a language REPL (Read-Evaluate-Print Loop), allowing you to evaluate pure expressions and inspect the results.
 
+Checking Coverage
+-----------------
+
+When ``daml test`` runs, it analyzes the ledger record to produce a report on what percentage of templates were created and which interface and template choices were exercised during our tests.
+
+Define Templates, Choices, and Interfaces
```

Comment:
```suggestion
Define templates, choices, and interfaces
```

Reply:
Are we still going with the difference in capitalization between titles? Last time I spoke with @carrielaben-da we agreed that changing the title case based on the level is unnecessarily complex.

Reply:
This is in our style guide. H1 and H2 should be capitalized, everything else sentence case. https://docs.google.com/document/d/1FEQP5r6XylrXinezP_k2FOBrTMnd_2YsTWVRHh0UOoE/edit

Reply:
Yeah, this is one of several items in the documentation standards that I'd like to review for simplicity and consistency.